deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -104,6 +114,20 @@
 
   @Override
   public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+    try {
+      processInternal(annotations, roundEnv);
+    } catch (DeepLinkProcessorException e) {
+      error(e.getElement(), e.getMessage());
+    }
+    return true;
```

Comment:
why did you change this to return true? I think in general it is best to return false so other processors can use the annotations too

Reply:
A fair point. I'll change it back.

My original reasoning to set this to false is that if you make your processor incremental (isolating), and another processor relies on your annotation as well, it could potentially rerun your processor as well.

But considering this is an aggregating processor, it doesn't really matter.